### ISSUE TITLE
Different chmod file/folders in SFTP

### ISSUE THREAD

**Issue by JMCA2** [mkloubert/vscode-remote-workspace#9]
Is it possible to set a different default mode for folders (such as 755) and files (such as 644)?

I didn't find the option.

Also, what's the default for creating new files?

**Comment by mkloubert** [comment]
@JMCA2 

No, this feature wasn't implemented yet.

**I have released a new version 0.25.0**, where I have added the parameter `dirMode`, which works the same way as `mode`. If not defined, `mode` is used, of course.

Example:

```
sftp://user:password@sftp.example.com/?dirMode=755&mode=644
```

**Comment by JMCA2** [comment]
Great. Thank you!
